app/v1/connectors/httpparsers/StandardDesHttpParser.scala

```diff
@@ -58,7 +58,7 @@ object StandardDesHttpParser extends HttpParser {
 
     response.status match {
       case successCode.status =>
-        logger.info(
+        logger.warn(
```

Comment:
This should be logged at info level and the error response should be logged at warn level